networking/hardware_networks/about-sriov.adoc

```diff
@@ -79,12 +94,6 @@ The SR-IOV Network resources injector and SR-IOV Network Operator webhook are en
 Use caution when disabling the SR-IOV Network Operator Admission Controller webhook. You can disable the webhook under specific circumstances, such as troubleshooting, or if you want to use unsupported devices.
 ====
 
-// Supported platforms
-include::modules/nw-sriov-supported-platforms.adoc[leveloffset=+2]
```

Comment:
The catalog does not provide this info. It only says if SR-IOV is supported in OCP. OCP can be installed in multiple platforms (e.g. bare metal, OpenStack, AWS, GCP, etc).